okhttp/src/main/java/okhttp3/internal/connection/RealConnection.kt

```diff
@@ -736,6 +729,7 @@ class RealConnection(
   companion object {
     private const val NPE_THROW_WITH_NULL = "throw with null exception"
     private const val MAX_TUNNEL_ATTEMPTS = 21
+    private const val IDLE_CONNECTION_HEALTHY_NS = 1_000_000_000 // 10 seconds.
```

Comment:
Isn't this one second?

Reply:
Endless screaming.

It is. I wanted to use const which took away TimeUnit.toNanos().